Projects/App/Sources/Screens/UpdateAppScreen/UpdateAppView.swift

```diff
@@ -0,0 +1,46 @@
+import Flow
+import Form
+import Foundation
+import Presentation
+import SwiftUI
+import UIKit
+import hCore
+import hCoreUI
+import hGraphQL
+
+struct UpdateAppView: View {
+    var body: some View {
+        ZStack {
```

Comment:
Maybe we should just use hForm and hRow here, so that this screen isn't too custom, what do you think?

Reply:
Agree 👍